client/my-sites/email/email-providers-comparison/price/discount-or-offer-terms.scss

```diff
@@ -0,0 +1,11 @@
+.discount-or-offer-terms__item {
+	margin-bottom: 12px;
+}
+
+.discount-or-offer-terms__discounted_item.badge--info-green {
+	background-color: rgba( 184, 230, 191, 0.24 );
+	border: solid 1px var( --studio-green-30 );
+	color: var( --studio-green-50 );
+	margin-left: 5px;
```

Comment:
> Currently, the margin is rendered even when the discounted badge is rendered as the very first item.

That was my intention, since it essentially distinguishes displays for offers vs displays for sale coupons. 
After thinking more about this, I realise that detail may _not_ be important to the user, as long as they are getting a discount.

I'll go with your suggestion because in the long run the UX seems better.